templates/common/powervs/units/powervs-hostname.service.yaml

```diff
@@ -11,8 +11,8 @@ contents: |
   Before=node-valid-hostname.service
 
   [Service]
-  ExecStart=/usr/bin/afterburn --provider powervs --hostname=/etc/hostname
   Type=oneshot
+  ExecStart=/usr/local/bin/mco-hostname --powervs
 
   [Install]
   WantedBy=network-online.target
```

Comment:
Best to update to this to:
```
WantedBy=node-valid-hostname.service
```
as it's a direct dependency of node-valid-hostname.